.devcontainer/devcontainer.json

```
@@ -1,19 +1,19 @@
 {
   "name": "Gitea DevContainer",
-  "image": "mcr.microsoft.com/devcontainers/go:1.24-bookworm",
+  "image": "mcr.microsoft.com/devcontainers/go:1.25-trixie",
   "containerEnv": {
     // override "local" from packaged version
     "GOTOOLCHAIN": "auto"
   },
   "features": {
     // installs nodejs into container
     "ghcr.io/devcontainers/features/node:1": {
-      "version": "lts"
+      "version": "24"
```

```suggestion
      "version": "latest"
```

https://github.com/devcontainers/features/blob/c05bd451518d73c9658eb9a41db18aaa968b73f3/src/node/install.sh#L295-L296